VVResonances/interactive/Projections3DHisto.C

```diff
@@ -281,7 +288,7 @@ for(int i=0; i<5; ++i){
  pullsz[i]->SetMarkerSize(0);
      
 }
-
+cy->SaveAs(Form("ProjectionsY_%s.pdf",sample));
```

Comment:
This could be removed as cy is already saved above

https://github.com/daniSchaefer/cmgtools-lite/blob/f0c4a6995cdfbfa675b744a7c2fc1ac639ffaf9c/VVResonances/interactive/Projections3DHisto.C#L139

Then I guess you can remove this line too

https://github.com/daniSchaefer/cmgtools-lite/blob/f0c4a6995cdfbfa675b744a7c2fc1ac639ffaf9c/VVResonances/interactive/Projections3DHisto.C#L22

If you want to add a label (for instance "herwig" or "pythia") you can pass a different output folder to the macro.